From 8e381c52125a6f1468313db257aafe5e03cd4753 Mon Sep 17 00:00:00 2001
From: Mario Schuknecht <mario.schuknecht@dresearch-fe.de>
Date: Wed, 8 Jul 2020 06:49:07 +0200
Subject: [PATCH] mke2fs.conf: define a short proceed_delay of 1 sec, HYP-14823

Beginning with this version mke2fs asks the user for confirmation if
block device or image file contains an existing file system image.
To avoid the necessary interaction in our scripts we define a short
proceed delay of just a second after which mke2fs continues
automatically.

Signed-off-by: Steffen Sledz <sledz@dresearch-fe.de>
---
 misc/mke2fs.conf.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/misc/mke2fs.conf.in b/misc/mke2fs.conf.in
index 00afd912..c95f55bb 100644
--- a/misc/mke2fs.conf.in
+++ b/misc/mke2fs.conf.in
@@ -48,3 +48,4 @@
 
 [options]
 	fname_encoding = utf8
+	proceed_delay = 1
-- 
2.16.4

